src/pages/aboutPage/About.tsx

```diff
@@ -1,56 +1,57 @@
-import 'semantic-ui-css/semantic.min.css'
+import "semantic-ui-css/semantic.min.css";
 import React, { useState } from "react";
-import { Button } from 'semantic-ui-react'
+import { Button } from "semantic-ui-react";
 import "./About.css";
-import teamImage from "../../images/team-image.svg"
-import Navbar from '../../components/navBar/Navbar';
+import teamImage from "../../images/team-image.svg";
+import Navbar from "../../components/navBar/Navbar";
 import { useHistory } from "react-router-dom";

```

Comment:
I ran this locally and it didn't like this import. I think it may have to be `import Navbar from "../../components/Navbar/Navbar"` Can you check it out locally and see which import is correct?